docs/reference.rst

```diff
@@ -1631,13 +1631,13 @@ RelatedFactory
         - Or the fully qualified path to a :class:`Factory` subclass
           (see :ref:`subfactory-circular` for details)
 
-    .. attribute:: name
+    .. attribute:: factory_related_name
 
         The generated object (where the :class:`RelatedFactory` attribute will
```

Comment:
Maybe we should reword that sentence too, I believe it should be:

```rst
The generated object (produced by the factory where the :class:`RelatedFactory` attribute is declard) may be passed to the related factory if the :attr:`factory_related_name` parameter is set.

Reply:
Updated with my own wording:

> If set, the object generated by the factory declaring the `RelatedFactory` is passed as keyword argument to the related factory.

Also, passed the `factory_related_name` as keyword argument instead of positional.

Let me know what you think.

Reply:
I'm ambivalent on the kwarg part: the typical use (in our tests, and in many codebases I've seen) is to pass it as a positional argument — see https://github.com/search?q=RelatedFactory&type=Code for some examples in the wild.

The rest sounds good to me, though :)

Reply:
It is declared as a kwarg in `RelatedFactory`, I think it is preferable to document a usage that matches the declaration.
Passing the keyword argument as a positional is also supported by Python, but passing it as a keyword argument should be preferred.
WDYT?

Reply:
I do not mind updating existing instances in our codebase for consistency, if that helps?

Reply:
Added commit `Prefer kwarg for RelatedFactory.factory_related_name` to that end.